DEPENDENCIES.md

```diff
@@ -91,8 +91,8 @@ sudo apt remove eosio
 #### RPM Package Install
 
 ```sh
-wget https://github.com/dfuse-io/eos/releases/download/v2.0.5-dm-12.0/eosio-2.0.5-dm.12.0-1.el7.x86_64.rpm
-sudo yum install ./eosio-2.0.5-dm.12.0-1.el7.x86_64.rpm
+wget curl -s https://api.github.com/repos/dfuse-io/eos/releases/latest|grep "browser_download_url.*.rpm"|cut -d : -f 2,3|tr -d \"|wget --show-progress -O ./eosio-dm-latest.rpm -qi -
```

Comment:
This one seems to be broken (`wget curl -s ...`)

Reply:
Oops, deleted initial wget (was editing live)